tests/test_base.py

```diff
@@ -215,6 +215,17 @@ class Meta:
         with self.assertRaises(TypeError):
             type("SecondFactory", (base.Factory,), {"Meta": Meta})
 
+    def test_meta_model_as_path(self):
+        class MailboxFactory(base.Factory):
+
+            class Meta:
+                model = "mailbox.Mailbox"
+            path = "/tmp/mail"
+
+        import mailbox
+        box = MailboxFactory()
+        assert isinstance(box, mailbox.Mailbox)
```

Comment:
[very optional] Maybe test that the variable was set correctly?

```suggestion
        assert isinstance(box, mailbox.Mailbox)
        assert box._path == "/tmp/mail"
```